packages/vue-app/template/App.js

```diff
@@ -277,9 +282,54 @@ export default {
         layout = 'default'
       }
       return Promise.resolve(layouts['_' + layout])
-    }
+    },
     <% } /* splitChunks.layouts */ %>
     <% } /* features.layouts */ %>
+    <% if (isFullStatic) { %>
+    setPagePayload(payload) {
+      this._pagePayload = payload
+      this._payloadFetchIndex = 0
+    },
+    getPayloadPath(href) {
+      return (`${this.payloadPath}/${href.replace(/\/$/, '')}/payload.json`).replace(/\/+/g, '/')
+    },
+    hasPayload(href) {
+      return !!(this._payloadCache && this._payloadCache[this.getPayloadPath(href)])
+    },
+    async fetchPayload (href) {
+      this._payloadCache = this._payloadCache || {}
+      const payloadPath = this.getPayloadPath(href)
+      let payload = this._payloadCache[payloadPath]
+
+      // If payload is a promise, returns it
+      if (payload && payload instanceof Promise && typeof payload.then === 'function') {
```

Comment:
`then` is promise proto. I think `instance of` a check is extra :D

Reply:
You can update, I was using old code for this check :D